src/utils.js

```diff
@@ -8,3 +9,9 @@ export const refreshList = (list, func) => {
     fetchData()
   }, list)
 }
+
+export const pageNav = async (func, route) => {
+  const navigate = useNavigate()
```

Comment:
this util may not work, since it uses a hook outside of a component or custom hook. If this meant to be a hook, consider naming it using the `use*` naming convention. 

Refer to rules on using hooks:
https://reactjs.org/docs/hooks-rules.html